README.md

```diff
@@ -6,6 +6,10 @@ Here you will find tutorials and interactive notebooks for ArangoDB features. Th
 
 ## Learn
 
+### Machine Learning
+In addition to the notebooks listed below we have many example machine learning notebooks for each project in the [ArangoML repositories](https://github.com/arangoml).
```

Comment:
Is it worth adding a link to the Arangopipe example notebooks directory here? (https://github.com/arangoml/arangopipe/tree/master/examples)

Reply:
Good idea, I have updated it. WDYT?

Reply:
Looks good to me! 👍